server/controllers/events/testdata/test-repos/import-multiple-project/dir1/main.tf

```diff
@@ -1,4 +1,3 @@
 resource "random_id" "dummy1" {
```

Comment:
Since this issue came from a provider version update, to keep the tests deterministic, what do you think of pinning the provider versions for each of these terraform random provider (and possibly other provider) tests?

Reply:
I agree we should probably pin this, or at least worth a discussion. I'm fine making that a followup and merging this as-is, since unit tests being broken on main is causing downstream issues.